project-lifecycle.md

```diff
@@ -118,3 +108,15 @@ period, the project will be labeled *End of Life*.
 # End of Life
 
 A project that is no longer actively developed or maintained.
+
+# Promoted Release
+
+A project\'s maintainers seeking to publish a *Promoted
+Release* (see Hyperledger\'s use of [semver](./release-taxonomy.md#1);
+also, [semver.org](https://semver.org)) must seek approval of the TSC
+whether in *Graduated* or *Incubation* status as defined above. While it is
```

Comment:
This section needs to be remove Incubation since the criteria-for-promoted-release.md says that the project must be in Graduated status.

Reply:
I merely moved that text but I agree it's a good opportunity to fix it.

Reply:
Ok, this too is leftover from First Major Release which did allow in Incubation.

Reply:
Would it help if we add a note here that would redirect the reader, i.e.

```
Projects seeking to have a *promoted release* must meet the eligibility criteria defined in the [Promoted Release](./criteria-for-promoted-release.md) document.
```

Capture the special statute of `Promoted Release` under `Incubation` in the same document where the criteria is captured.

In this section, the term `Promoted Release` must be defined (along with any benefits of doing that). For example, the section `Incubation` tells clearly what benefits do each projects receive. I see that all other sections in this page are defining what the term means. But I couldn't find a definition for `Promoted Release`.

Reply:
It is a fact that "Promoted Release" isn't currently defined anywhere but I don't think it belongs here. This page defines the stages a project can be in. Promoted Release is not such a stage. I think we should look into expanding on the page linked from this document to not only define the criteria for a promoted release but also defining what a promoted release actually is.

Reply:
Sure, thanks.